pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle/ecdhes_aes_aead_composite_test.go

```diff
@@ -87,6 +88,14 @@ func TestEncryptDecryptNegativeTCs(t *testing.T) {
 
 	mEncHelper.KeySizeValue = 32
 
+	// Encrypt should fail with bad key type
+	cEnc.keyType = ecdhespb.KeyType_UNKNOWN_KEY_TYPE
+
+	_, err = cEnc.Encrypt(pt, aad)
+	require.Error(t, err)
```

Comment:
Should you be checking for certain error type/message?

Reply:
added